README.md

```diff
@@ -3,35 +3,94 @@
 </br>
 </br>
 
-<div align="center">
+<p align="center">
+  <img src="assets/logo.png" alt="xk6-disruptor" width="300"></a>
+  <br>
+  "Like Unit Testing, but for <strong>Reliability</strong>"
+  <br>
+</p>
+<p align="center">  
+</div>
 
-![logo](assets/logo.png)
 
+xk6-disruptor is an extension adds fault injection capabilities to [Grafana k6](https://github.com/grafana/k6). It implements the ideas of the Chaos Engineering discipline and enables Grafana k6 users to test their system's reliability under turbulent conditions.
 
-</div>
+<blockquote>
+⚠️ <strong>Important</strong> ⚠️ 
+<br>This project is still in its early stages and under active development.
+<br>The API is subject to change, and there may be bugs.
+<br>Use at your own risk. Thanks!
```

Comment:
```suggestion
> ⚠️  xk6-disruptor is in the alpha stage, undergoing active development. We do not guarantee API compatibility between releases - your k6 scripts may need to be updated on each release until this extension reaches v1.0 release.
```

Reply:
@dgzlopes In an early version of the README we decided that "use at your own risk" may deter people form using the tool. We prefer this wording.

Reply:
Sounds good!